### PR TITLE
Add TMB Docker

### PR DESCRIPTION
**Pull request by sickler-alex**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings
- [X] I have committed any related changes to the PR

### CODE REVIEW

File: toolkit_tmb/1.0.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM rocker/tidyverse:4.2.1
+LABEL maintainer="Alex Sickler <sicklera@chop.edu>"
+LABEL description="tidyverse image with tmb 1.0.0 package"
+
+ENV OPTPARSE_VERSION 1.7.3
+ENV BIOC_VERSION 3.16
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```
Comment (dmiller15) [inline toolkit_tmb/1.0.0/Dockerfile:10]: What is the purpose of this pipefail?

**Review by migbro — APPROVED** [review]
Passes circici build, but also curious about the pipefail inclusion, even though `&&` is used...

File: toolkit_tmb/1.0.0/Dockerfile (hunk 1 above)
Comment (dmiller15) [reply toolkit_tmb/1.0.0/Dockerfile:10]: Yeah it doesn't appear to be doing anything here. Best to get rid of it.

Comment (sickler-alex) [reply toolkit_tmb/1.0.0/Dockerfile]: Done.

**Review by dmiller15 — APPROVED** [review]

**Pull request merged**